### PR TITLE
Created database and tables for Animals and thier properties

### PR DESCRIPTION
**Pull request by mohashyne**
# New Features added

- [ ] Add readme.md file and MIT.md license file
- [ ] Created a database named vet_clinic.
- [ ] Created a table named animals
- [ ] Insert data ( animals and their properties)
- [ ] Run/store queries for the quests;

#  Some query quests  and screenshots

- [ ] SELECT * FROM animals WHERE name LIKE '%mon'; 
end_with_mon

- [ ] SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';  
select_birth_date

- [ ] SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
name_neu_escape

- [ ] SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu'); 
select_birth-OR

- [ ] SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5; 
name_ecape-attempts

- [ ] SELECT * FROM animals WHERE neutered = true;
neutered

- [ ] SELECT * FROM animals WHERE name <> 'Gabumon';  
NOT_gabamon

- [ ] SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3; 
weight_btween

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (usorfaitheloho) [inline queries.sql:7]: According to the requirement, you are to select name and not * from animals, kindly make the correction before submitting again 😄 

image

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+id BIGSERIAL NOT NULL PRIMARY KEY,
+name VARCHAR(300) NOT NULL,
+date_of_birth DATE NOT NULL,
+escape_attempts INTEGER,
+neutered BOOLEAN,
+weight_kg DECIMAL );
```
Comment (usorfaitheloho) [inline schema.sql:9]: Please note that you are to add the screenshots of your query results to the pull request description, so as to see if the results are correct. This also helps to ensure that you run the queries with success 👍 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mohashyne  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All queries were written ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (mohashyne) [reply schema.sql:9]: Of course I did add the links for running the queries, did you click on the link to see the screenshots at all ?

**Review by algerina — APPROVED** [review]
Hi @mohashyne ,

Great job so far!  

May I highlight the following: 
- [x]  You have a descriptive and professional Pull request👏  ✔️ 
- [x] You code is clean and well organized✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**